cpp/include/raft/neighbors/detail/cagra/cagra_search.cuh

```diff
@@ -243,6 +294,9 @@ void search_main(raft::resources const& res,
     RAFT_FAIL("Unrecognized dataset format");
   }
 
+  // Refinement
+  if (use_vpq) {}
+
```

Comment:
Refinement in raft is a separate operation, no need to include it into cagra-q search or parameters.